api/src/main/java/org/openmrs/module/insuranceclaims/Item.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.insuranceclaims;
+
+import org.openmrs.BaseOpenmrsData;
+import org.openmrs.User;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+/**
+ * Please note that a corresponding table schema must be created in liquibase.xml.
+ */
+//Uncomment 2 lines below if you want to make the Item class persistable, see also InsuranceClaimsDaoTest and liquibase.xml
+//@Entity(name = "insuranceclaims.Item")
+//@Table(name = "insuranceclaims_item")
+public class Item extends BaseOpenmrsData {
+
+	private static final long serialVersionUID = 8352049103901464406L;
```

Comment:
why we need to add serialVersionUID ? BaseOpenMRSdata uses serializable interface ?

Reply:
Yes, we should. BaseOpenmrsData extends the BaseOpenmrsObject which uses serializable interface. In the separate PR where I will add all requires POJO classes I will add this interface for each class then it should increase readability.